src/axom/quest/util/mesh_helpers.hpp

```diff
@@ -78,22 +78,52 @@ mfem::Mesh* make_cartesian_mfem_mesh_3D(const primal::BoundingBox<double, 3>& bb
 
   \return The meshGrp pointer
 */
-axom::sidre::Group* make_structured_blueprint_box_mesh(
+axom::sidre::Group* make_structured_blueprint_box_mesh_3d(
   axom::sidre::Group* meshGrp,
   const primal::BoundingBox<double, 3>& bbox,
   const NumericArray<int, 3>& res,
   const std::string& topologyName = "mesh",
   const std::string& coordsetName = "coords",
   axom::runtime_policy::Policy runtimePolicy = axom::runtime_policy::Policy::seq);
 
-axom::sidre::Group* make_unstructured_blueprint_box_mesh(
+axom::sidre::Group* make_unstructured_blueprint_box_mesh_3d(
   axom::sidre::Group* meshGrp,
   const primal::BoundingBox<double, 3>& bbox,
   const NumericArray<int, 3>& res,
   const std::string& topologyName = "mesh",
   const std::string& coordsetName = "coords",
   axom::runtime_policy::Policy runtimePolicy = axom::runtime_policy::Policy::seq);
 
+/*!
+  /brief Creates a 2D Cartesian blueprint mesh lying within a given bounding box
+
+  \param meshGrp Put the mesh in this Group
+  \param bbox The bounding box for the mesh
+  \param res The resolution of the mesh
+  \param topologyName Name of the blueprint topoloyy to use.
+  \param coordsetName Name of the blueprint coordset to use.
+  \param runtimePolicy Runtime policy, see axom::runtime_policy.
+         Memory in \c meshGrp must be compatible with the
+         specified policy.
+
+  \return The meshGrp pointer
+*/
+axom::sidre::Group* make_structured_blueprint_box_mesh_2d(
+  axom::sidre::Group* meshGrp,
+  const primal::BoundingBox<double, 2>& bbox,
+  const NumericArray<int, 2>& res,
+  const std::string& topologyName = "mesh",
+  const std::string& coordsetName = "coords",
+  axom::runtime_policy::Policy runtimePolicy = axom::runtime_policy::Policy::seq);
+
+axom::sidre::Group* make_unstructured_blueprint_box_mesh_2d(
+  axom::sidre::Group* meshGrp,
+  const primal::BoundingBox<double, 2>& bbox,
+  const NumericArray<int, 2>& res,
+  const std::string& topologyName = "mesh",
+  const std::string& coordsetName = "coords",
+  axom::runtime_policy::Policy runtimePolicy = axom::runtime_policy::Policy::seq);
```

Comment:
This function is declared identically twice.

Reply:
These are both necessary, one is for `unstructured` and the other for `structured` mesh generation (though I agree the names are very similar).